Jenkinsfile

```diff
@@ -906,7 +913,7 @@ def run_pytorch_tests(Map conf=[:]){
     }
 
     withDockerContainer(image: image, args: dockerOpts) {
-        timeout(time: 45, unit: 'MINUTES'){
+        timeout(time: 2, unit: 'HOURS'){
```

Comment:
It would be helpful to have a comment about why this value was chosen and what an engineer should consider if changing it. What are the pros and cons for large or small values?

Reply:
Hi John, yes, good point. The timeout is set for each CI stage individually. The rule of the thumb is that it should be sufficient for all the tests to finish, but not too generous that in case something hangs, we're going to waste too much machine time before killing the job. Previously, the tests in that stage only took ~10-12 minutes + 7-10 minutes to pull the docker image. So 45 was ample. With more tests added, it became too tight, so I had to increase the timeout to 2 hours.